reconcile/utils/terraform_client.py

```diff
@@ -159,6 +168,7 @@ def log_plan_diff(self, name, tf, enable_deletion):
         # or if the integration's enable_deletion is true
         deletions_allowed = enable_deletion or account_enable_deletion
         deleted_users = []
+        created_users = []
```

Comment:
Are these users already created or to be created at a different point of the integration?

Reply:
we are in the context of `terraform plan`, so they are not yet created, but they are as good as created.